__tests__/index.test.js

```diff
@@ -22,28 +24,86 @@ describe("GET /api/topics", () => {
       });
   });
 });
-
+describe("Error: invalid_path", () => {
+  it("GET: 404 should respond with an error when the url is incorrect", () => {
+    return request(app)
+      .get("/api/twopics")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Invalid path");
+      });
+  });
+});
 describe("GET /api", () => {
   it("GET: 200 should respond with an object describing all the available endpoints on our API", () => {
     return request(app)
       .get("/api")
       .expect(200)
       .then((response) => {
-        const expected = ["GET /api", "GET /api/topics", "GET /api/articles"];
+        const expected = endpoints
         const { body } = response;
-        const actual = Object.keys(body);
-        expect(actual).toMatchObject(expected);
+        expect(body).toMatchObject(expected);
       });
   });
 });
-
-describe('Error: invalid_path', () => {
-    it("GET: 404 should respond with an error when the url is incorrect", () => {
-        return request(app)
-          .get("/api/twopics")
-          .expect(404)
-          .then((response) => {
-            expect(response.body.msg).toBe("Invalid path");
-          });
+describe("GET /api/articles/:article_id", () => {
+  it("GET 200: should respond with an article object with the following attributes: author, title, article_id, body, topic, created_at, votes, article_img_url", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const expected = {
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        };
+        const { body } = response;
+        const actual = body.article;
+        expect(actual).toMatchObject(expected);
```

Comment:
Great use of `.toMatchObject` for a flexible test that will still pass if you add extra keys to the response later